lib/devicetrust/native/others.go

```diff
@@ -0,0 +1,36 @@
+// TODO(codingllama): Tweak build tag above once we have the darwin impl.
+//go:build darwin || !darwin
+
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package native
+
+import (
+	"errors"
+
+	devicepb "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1"
+)
+
+func enrollDeviceInit() (*devicepb.EnrollDeviceInit, error) {
+	return nil, errors.New("not implemented")
```

Comment:
```suggestion
	return nil, trace.NotImplemented("platform not supported")
```

or something along those lines.

Reply:
I've been purposefully avoiding trace.NotImplemented because tsh seem to show some extra messages when it sees it. Also, I plan on using trace.NotImplemented to detect the absence of a server-side Device Trust impl (OSS Teleport / old Enterprise).

Changed messages and added a comment.